src/constructs/aws/Queue.ts

```diff
@@ -250,53 +254,93 @@ export class Queue extends AwsConstruct {
     async listDlq(): Promise<void> {
         const dlqUrl = await this.getDlqUrl();
         if (dlqUrl === undefined) {
-            console.log(
-                chalk.red(
-                    'Could not find the dead letter queue in the deployed stack. Try running "serverless deploy" first?'
-                )
+            throw new ServerlessError(
+                'Could not find the dead letter queue in the deployed stack. Try running "serverless deploy" first?',
+                "LIFT_MISSING_STACK_OUTPUT"
             );
-
-            return;
         }
-        const progress = ora("Polling failed messages from the dead letter queue").start();
+        let progressV2: Ora | undefined;
+        let progressV3: Progress | undefined;
+        if (isV3) {
+            progressV3 = progress.create({
+                message: "Polling failed messages from the dead letter queue",
+            });
+        } else {
+            progressV2 = ora("Polling failed messages from the dead letter queue").start();
+        }
         const messages = await pollMessages({
```

Comment:
maybe for readability, it would be easier to have 2 callbacks. One for v2 and one for v3

Reply:
I've gone over the whole PR to try and minimize the differences, but in some cases honestly it wasn't easy to do much better.